examples/async_jamba_tokenizer.py

```diff
@@ -0,0 +1,18 @@
+import asyncio
+
+from ai21_tokenizer import Tokenizer, PreTrainedTokenizers
+
+
+async def main():
+    tokenizer = await Tokenizer.get_async_tokenizer(PreTrainedTokenizers.JAMBA_INSTRUCT_TOKENIZER)
+
+    example_sentence = "This sentence should be encoded and then decoded. Hurray!!!!"
+    encoded = await tokenizer.encode(example_sentence)
```

Comment:
When calling the instance methods, the IDE isn't aware that object you're infact using, is async because you're returning `BaseTokenizer` which doesn't have async methods. Perhaps its not necessary to unite them under the same class and have async and sync separately.
  
![image](https://github.com/AI21Labs/ai21-tokenizer/assets/39553475/734dbed1-274c-4470-b61f-fae888a8c578)